includes/cleanup.php

```diff
@@ -39,32 +34,37 @@ function pmpro_delete_user_form_notice( $current_user, $userids ) {
 	// Show a notice if users for deletion have an an active membership level.
 	if ( ! empty( $userids_have_levels ) ) { ?>
 		<div class="notice notice-error inline">
-			<?php if ( count( $userids ) > 1 ) {
+			<?php
+			if ( count( $userids ) > 1 ) {
 				_e( '<p><strong>Warning:</strong> One or more users for deletion have an active membership level. Deleting a user will also cancel their membership and recurring subscription.</p>', 'paid-memberships-pro' );
 			} else {
 				_e( '<p><strong>Warning:</strong> This user has an active membership level. Deleting a user will also cancel their membership and recurring subscription.</p>', 'paid-memberships-pro' );
 			}
-		?>
+			?>
 		</div>
-	<?php
+		<?php
 	}
 }
 add_action( 'delete_user_form', 'pmpro_delete_user_form_notice', 10, 2 );
 
-//deleting a category? remove any level associations
-function pmpro_delete_category($cat_id = NULL)
-{
+// deleting a category? remove any level associations
+function pmpro_delete_category( $cat_id = null ) {
 	global $wpdb;
-	$sqlQuery = "DELETE FROM $wpdb->pmpro_memberships_categories WHERE category_id = '" . $cat_id . "'";
-	$wpdb->query($sqlQuery);
+	$sqlQuery = $wpdb->prepare(
+		"DELETE FROM $wpdb->pmpro_memberships_categories WHERE category_id = '%d'",
```

Comment:
We don't use quotes anywhere for placeholders when using prepare, we just pass in %d or %s

```suggestion
		"DELETE FROM $wpdb->pmpro_memberships_categories WHERE category_id = %d",
```

Reply:
Moved over to using the delete method instead.